### ISSUE TITLE
Poor Results In Pancreas

### ISSUE THREAD

**Issue by YonghanLU** [JuliaWolleb/Diffusion-based-Segmentation#30]
When I trained the model you proposed on the pancreatic dataset, I found that the performance was poor, but the loss function converged quickly. I extracted the slices from the .nii files and converted them to .npy files, and finally transformed them into tensors to feed into the diffusion model. Have you ever encountered this issue?

**Comment by JuliaWolleb** [comment]
Hi
The loss function indeed converges very quickly, but still you need to go on with the training for a while (this is the case for all diffusion models that I trained so far). For how many epochs/iterations did you train? Does the performance get better when you train longer?

**Comment by YonghanLU** [comment]
Than you for your response,I will try to train longer .
